Assets/Project/Modules/Enemies/Turret/Scripts/ParabolicTurret/AreaDamageOverTime.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Popeye.Core.Pool;
+using Popeye.Core.Services.ServiceLocator;
+using Popeye.Modules.CombatSystem;
+using UnityEngine;
+
+public class AreaDamageOverTime : RecyclableObject
+{
+    
+    private DamageHit _contactDamageHit;
+    [SerializeField] private DamageHitConfig _contactDamageConfig;
+
+    private ICombatManager _combatManager;
+
+    private bool _standingOnArea = false;
+    private GameObject _player;
+
+    [SerializeField] private float _burnRate = 0.3f;
+    [SerializeField] private float _lifeTime = 0.3f;
+    private float burnTimer = 0;
```

Comment:
From now on, whenever you need to make a timer, use the Timer class I created. it is in the module "Popeye.Modules.Utilities"
https://github.com/TomeuGarcia/PopeyeTFG/blob/feature/reworkTurret/Assets/Project/Modules/Utilities/Scripts/Timer.cs